src/Maestro/DependencyUpdateErrorProcessor/DependencyUpdateErrorProcessor.csproj

```diff
@@ -28,6 +28,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\DotNet.Status.Web\DotNet.Status.Web.csproj" />
```

Comment:
Don't reference the entire web project.  This needs to get moved to a different assembly. Probably the current GitHub.Authentication one. That's where it makes the most sense.

Reply:
Removed this reference and moved GithubClientFactory to Microsoft.DotNet.GitHub.Authentication proj